packages/core/components/MenuBar/styles.module.css

```diff
@@ -0,0 +1,46 @@
+.MenuBar {
+  background-color: var(--puck-color-white);
+  border-bottom: 1px solid var(--puck-color-grey-8);
+  display: none;
+  left: 0;
+  margin-top: 1px;
+  padding: 8px 16px;
+  position: absolute;
+  right: 0;
+  top: 100%;
+  z-index: 1;
+}
+
+.MenuBar--menuOpen {
+  display: block;
+}
+
+@media (min-width: 638px) {
+  .MenuBar {
+    border: none;
+    display: block;
+    margin-top: 0;
+    overflow-y: visible;
+    padding: 0;
+    position: static;
+  }
+}
+
+.MenuBar-inner {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px 16px;
+  justify-content: flex-end;
+}
```

Comment:
We should also add `align-items: center`, so the use can add items of any size using `renderHeaderActions`

(fwiw, our secondary Button is also not rendering at the same height as the primary one, which is how I noticed this)

Reply:
Done